docs/rst/getting_started/tutorial.rst

```diff
@@ -6,6 +6,69 @@
 Example of usage
 ################
 
-This example will show how to monitoring a DDS network using *eProsima DDS Visualizer Plugin* and how to
-understand the different plugin features and configurations.
+This example will show how to monitor a DDS network using *eProsima DDS Visualizer Plugin*, by leveraging some of the
+features and configuration options this add-on has to offer.
 
+We will make use of `ShapesDemo <https://www.eprosima.com/index.php/products-all/eprosima-shapes-demo>`_, a tool
+primarily utilized for testing interoperability between different DDS implementations, as well as to experiment with
+different QoS configurations. Shapes of different types and colors move around the screen, while publishing their
+coordinates in real time over DDS.
+
+Let us first launch an instance of *ShapesDemo*, and start publishing under the ``Square`` topic.
+Choose the DDS domain on which data will be published (Options->Participant Configuration), and create a publisher with
+default settings on this topic.
+
+|domain| |settings| |square|
+
+.. |domain| image:: /rst/figures/usage_example/shapesdemo_domain_selection.png
+   :width: 33%
+
+.. |settings| image:: /rst/figures/usage_example/publisher_settings.png
+   :width: 33%
+
+.. |square| image:: /rst/figures/usage_example/square_shape.png
+   :width: 33%
+
+Run now *Plotjuggler* application, and start streaming data through the |ddsvisualizer|. In the DDS configuration
```

Comment:
I would separate each step in different subsections (or subsubsections or whatever).
And add an index at the top so the page is easier to navigate.
Like this it is difficult to follow as there are big pictures with white background and small text paragraphs.